cypress/support/commands/form.js

```diff
@@ -266,89 +266,78 @@ Cypress.Commands.add(
     skipRequest = false
   ) => {
-    describe('Enter value into lookup list field', function() {
-      let path = `#lookup_${fieldName}`;
-      if (modal) {
-        path = `.panel-modal ${path}`;
-      }
+    let path = `#lookup_${fieldName}`;
+    if (modal) {
+      path = `.panel-modal ${path}`;
+    }
 
-      const aliasName = `writeIntoLookupListField-${fieldName}-${new Date().getTime()}`;
-      //the value to wait for would not be e.g. "Letter", but {key: "540408", caption: "Letter"}
-      const expectedPatchValue = removeSubstringsWithCurlyBrackets(partialValue);
-      // in the default pattern we want to match URLs that do *not* end with "/NEW"
-      const patchUrlPattern = rewriteUrl || '/rest/api/window';
-      if (!skipRequest) {
-        cy.server();
-        cy.route('PATCH', new RegExp(patchUrlPattern)).as(aliasName);
+    const aliasName = `writeIntoLookupListField-${fieldName}-${new Date().getTime()}`;
+    //the value to wait for would not be e.g. "Letter", but {key: "540408", caption: "Letter"}
+    const expectedPatchValue = removeSubstringsWithCurlyBrackets(partialValue);
+    // in the default pattern we want to match URLs that do *not* end with "/NEW"
+    const patchUrlPattern = rewriteUrl || '/rest/api/window';
+    if (!skipRequest) {
+      cy.server();
+      cy.route('PATCH', new RegExp(patchUrlPattern)).as(aliasName);
+    }
+    cy.get(path).within(el => {
+      if (el.find('.lookup-widget-wrapper input').length) {
+        return (
+          cy
+            .get('input')
+            // we can't use `clear` here as sometimes it triggers request to the server
+            // and then the whole flow becomes flaky
+            .type('{selectall}')
+            .type(partialValue)
+        );
       }
-      cy.get(path).within(el => {
-        if (el.find('.lookup-widget-wrapper input').length) {
-          return (
-            cy
-              .get('input')
-              // we can't use `clear` here as sometimes it triggers request to the server
-              // and then the whole flov becomes flaky
-              .type('{selectall}')
-              .type(partialValue)
-          );
-        }
-
-        cy.get('.lookup-dropdown').click();
-        return cy.get('.lookup-dropdown').click();
-      });
 
-      cy.get('.input-dropdown-list').should('exist');
-      cy.contains('.input-dropdown-list-option', expectedListValue).click(/*{ force: true }*/);
-      if (!skipRequest) {
-        cy.waitForFieldValue(`@${aliasName}`, fieldName, expectedPatchValue, typeList /*expectEmptyRequest*/);
-      }
-      cy.get('.input-dropdown-list .input-dropdown-list-header').should('not.exist');
+      // this is extremely fiddly when selecting from a combo field such as bpartner address.
+      // it will work locally, but most of the times will fail in jenkins.
+      // Please create the tests such that adding an address in a combo field is not mandatory!
+      return cy.get('.lookup-dropdown').click();
```

Comment:
This is so annaying. First we didn't needed that, then we did, now it breaks sh*t again. I'm so fed up with this one.

Reply:
Yeah, tried even with 
```javascript
cy.get('.lookup-dropdown').dblclick().click();
```
and some other stuff. It's still hit and miss here.

I've just created this, maybe we'll have time some day to check wtf is going on (if it's cypress, if it's us, idk): https://github.com/metasfresh/metasfresh-e2e/issues/260